.github/workflows/build-wheels-linux.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Linux Wheels
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  generate-matrix:
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    with:
+      package-type: wheel
+      os: linux
+      test-infra-repository: pytorch/test-infra
+      test-infra-ref: main
+      with-cuda: disable
+      with-rocm: disable
+      with-cpu: enable
```

Comment:
no need to include with-cpu here, its enabled by default"
https://github.com/pytorch/test-infra/blob/main/.github/workflows/generate_binary_build_matrix.yml#L36